media-libs/gegl/gegl-0.2.0-r3.ebuild

```diff
@@ -82,6 +84,7 @@ src_prepare() {
 	[[ -f "${deps_file}" ]] || touch "${deps_file}"
 
 #	use vala && vala_src_prepare
+	eapply_user
```

Comment:
this is a poor attempt at porting an ebuild to EAPI 6. Users will not be able to patch `configure.ac`. This sounds like an attempt to make Portage not complain about missing `eapply_user`.

Reply:
What is a the proper way of fixing this for EAPI6? I've been seeing that method used in various overlays.

Reply:
Soap's point is the location of the call / the order of commands. If `eapply_user` goes _after_ (the existing) call to `eautoreconf`, patches against build system files like `configure.ac` will not propagate to `configure`. So `eautoreconf` should be called after `eapply_user`. r4 is calling them in that order:
```
	eapply_user
	eautoreconf
```